lib/getPostcssResult.js

```diff
@@ -19,7 +25,7 @@ module.exports = function (stylelint/*: stylelint$internalApi*/)/*: Promise<?Obj
     customSyntax?: string
   }*/ = arguments.length > 1 && arguments[1] !== undefined ? arguments[1] : {}
 
-  const cached = stylelint._postcssResultCache.get(options.filePath)
+  const cached/*: Object*/ = stylelint._postcssResultCache.get(options.filePath)
```

Comment:
yes, you are right, @davidtheclark , it's not just an Object, my bad... besides it could be `undefined` too. I'm trying to figure out the structure of new `postcss$result` for creating type declaration. Hope I'll resolve it